baseplate/context/redis.py

```diff
@@ -148,3 +151,62 @@ def __init__(self, trace_name, server_span, connection_pool,
     def execute(self, **kwargs):
         with self.server_span.make_child(self.trace_name):
             return super(MonitoredRedisPipeline, self).execute(**kwargs)
+
+
+class MessageQueue(object):
+    """A redis-backed variant of ``message_queue.MessageQueue``.
+
+    ``name`` can be any string.
```

Comment:
I will change it, but it is this same way for the `message_queue::MessageQueue` docstring (which is where I copied the style from).